hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -109,14 +109,19 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
 
     def __init__(self, market_pairs: List[CrossExchangeMarketPair], min_profitability: float,
                  trade_size_override: Optional[float] = 0.0,
+                 #adjust the size of taker volume for conservativeness
```

Comment:
current code style we use is to include a space between '#' and text. please fix all your comments

Reply:
also think its kind of strange to have comments in-between function variables, maybe easier to read if to right of the associate line?

Reply:
this is not even my file.

Reply:
let me use the current cross exchange market making file, i am not sure why this was committed